lib/pbench/test/unit/server/test_datasets_compare.py

```diff
@@ -0,0 +1,196 @@
+from http import HTTPStatus
+from pathlib import Path
+
+from pquisby.lib.post_processing import QuisbyProcessing
+import pytest
+import requests
+
+from pbench.server import JSON
+from pbench.server.cache_manager import CacheManager
+from pbench.server.database.models.datasets import Dataset, DatasetNotFound, Metadata
+
+
+def mock_get_value(dataset, key) -> str:
```

Comment:
`Metadata.getvalue()` takes three parameters, the last of which is optional, so the mock really should do the same.